packages/terra-clinical-result/src/proptypes/observationPropTypes.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import interpretationPropType from './interpretationPropTypes';
+import resultTypePropType from './resulttypePropTypes';
+import statusPropType from './statusPropTypes';
+import { valueQuantityPropShape, valueStringPropShape, valueNullPropShape } from './valuePropTypes';
+
+const observationPropShape = PropTypes.shape({
+  /**
+   *  Event ID for result
+   */
+  eventId: PropTypes.string,
+  /**
+   *  Value and optional Unit of Measure for the Observation Result
+   */
+  result: PropTypes.oneOfType([
+    valueQuantityPropShape,
+    valueStringPropShape,
+    valueNullPropShape,
+  ]),
+  /**
+   * Enum for possible Result Interpretation values (also called `Clinical Severity` and `Normalcy`).
+   * One of `'CRITICAL'`, `'EXTREMEHIGH'`, `'EXTREMELOW'`, `'PANICHIGH'`, `'PANICLOW'`, `'VABNORMAL'`, `'POSITIVE'`, `'ABNORMAL'`, `'HIGH'`, `'LOW'`, `'NORMAL'`, `'NEUTRAL'`.
+   */
+  interpretation: interpretationPropType,
+  /**
+   *  Enum for possible Result Types.
+   * One of `'ALPHA'`, `'MULTIALPHA'`, `'FREETEXT'`, `'NUMERIC'`, `'BLOODPRESSURE'`, `'CALCULATION'`, `'DATE'`, `'DATETIME'`, `'TIME'`, `'PROVIDER'`.
+   */
+  type: resultTypePropType,
+  /**
+   * Enum for possible Result Statuses.
+   * One of `'registered'`, `'preliminary'`, `'final'`, `'amended'`, `'corrected'`, `'cancelled'`, `'entered-in-error'`, `'unknown'`.
+   */
+  status: statusPropType,
+  /**
+   *  Clinical datetime for the Result
+   */
+  performedDateTime: PropTypes.string,
```

Comment:
Should this be required?

Reply:
not needed to be, part of additional result metadata that may be consumed by tooltip or additional component detail views/display not yet added (enhancements) .

Reply:
Based on the examples, is this expected to be in the ISO 8601 format? If so, should we note that in the prop documentation?